tests/unittests/utils/test_sapiens.py

```
@@ -0,0 +1,257 @@
+        )
+        with open(path) as f:
+            data = json.load(f)
+        return {m["base_name"]: m for m in data["models"]}
+
+    def test_all_four_registered(self, entries):
+        for name, _ in _POSE_MODELS:
+            assert name in entries, name
+        # The wrapper must support every registered repo.
+        assert set(fus._POSE_REPOS) == {repo for _, repo in _POSE_MODELS}
+
+    @pytest.mark.parametrize("name,repo", _POSE_MODELS)
```

Ruff (0.15.13)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)